opencog/openpsi/utilities.scm

```diff
@@ -326,13 +326,13 @@
 		result
 		; else check if entity is an evaluation or predicate or schema
 		(let ((type (cog-type entity)))
-			(if (or (equal? type 'PredicateNode)
-				   (equal? type 'GroundedPredicateNode)
+			(if (or (equal? type 'GroundedPredicateNode)
 				   (equal? type 'DefinedPredicateNode))
 				(set! result (cog-evaluate! (Evaluation entity (List)))))
-			(if (or (equal? type 'SchemaNode)
-            		(equal? type 'GroundedSchemaNode)
-            	    (equal? type 'DefinedSchemaeNode))
+			(if (equal? type 'PredicateNode)
+				(set! result (cog-tv entity)))
```

Comment:
I made the change because
```
scheme@(guile-user)> (cog-tv (EvaluationLink (Predicate "a" (stv .5 .5))))
$5 = (stv 1 0)
```
and that is what is done @ https://github.com/opencog/atomspace/blob/master/opencog/atoms/execution/EvaluationLink.cc#L253

I agree with Linas's suggestion of enforcing a single convention of a consistent syntax and semantics.